x-pack/plugins/apm/public/components/app/infra_overview/infra_tabs/use_tabs.test.tsx

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { renderHook } from '@testing-library/react-hooks';
+import { useTabs } from './use_tabs';
+
+describe('useTabs', () => {
```

Comment:
You could also mock the infra plugin and use a wrapper (`createKibanaReactContext`) to check that the methods are called / rendered correctly.